pkg/preflight/collect.go

```diff
@@ -131,6 +132,10 @@ func Collect(opts CollectOpts, p *troubleshootv1beta2.Preflight) (CollectResult,
 	collectSpecs = collect.EnsureCollectorInList(collectSpecs, troubleshootv1beta2.Collect{ClusterResources: &troubleshootv1beta2.ClusterResources{}})
 	collectSpecs = collect.EnsureClusterResourcesFirst(collectSpecs)
 
+	opts.KubernetesRestConfig.QPS = 100
```

Comment:
I'd create constants out of these magic numbers `const DEFAULT_CLIENT_QPS = 100` for example

```go
pkg
   constants
       constants.go
```

Reply:
Thank you @banjoh! I have added constants.go